docs/component-status.md

```diff
@@ -0,0 +1,37 @@
+| [Breadcrumb](/componentsbreadcrumb.md) | ![WCAG 2.0 Level AA](https://img.shields.io/badge/spec-aa-blue.svg) | ![Spec passing](https://img.shields.io/badge/spec-passing-brightgreen.svg) | ![Spec passing](https://img.shields.io/badge/spec-passing-brightgreen.svg)          |                                                                                                                                                              | ![Spec failing](https://img.shields.io/badge/spec-failing-red.svg)         |
+| [Button](/components/button.md)        | ![WCAG 2.0 Level A](https://img.shields.io/badge/spec-a-blue.svg)   | ![Spec passing](https://img.shields.io/badge/spec-passing-brightgreen.svg) | ![Spec passing](https://img.shields.io/badge/spec-passing-brightgreen.svg)          | ![Status new](https://img.shields.io/badge/status-new-blueviolet.svg)<br>![Spec passing with warnings](https://img.shields.io/badge/spec-warning-yellow.svg) | ![Spec failing](https://img.shields.io/badge/spec-failing-red.svg)         |
+
+## Community components
+
+|                                     | Specification                                                                                                                                            | Vanilla | React                                                                                                                                                         | Angular                                                                                                                                             | Vue |
+| ----------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| [Carousel](/components/carousel.md) | ![Status deprecated](https://img.shields.io/badge/status-deprecated-4c4c4c.svg)                                                                          |         | ![Status deprecated](https://img.shields.io/badge/status-deprecated-4c4c4c.svg)<br>![Spec passing](https://img.shields.io/badge/spec-passing-brightgreen.svg) |                                                                                                                                                     |     |
```

Comment:
What if a framework doesn't want to deprecate? Would the spec stay forever?

Reply:
I think we could come up deprecation logic at each major release similar to what @joshblack outlined here: https://github.com/carbon-design-system/rfcs/pull/5#issuecomment-484952979

We'll have to clearly document community component governance so any directory of community components doesn't become a graveyard of unsupported components.